packages/blade/src/components/Alert/_decisions/decisions.md

```diff
@@ -104,6 +104,17 @@ There'll also be an internal `BaseAlert` component that handles rendering web or
 
 ## a11y
 
+Web:
```

Comment:
Link to the APG pattern.
https://www.w3.org/WAI/ARIA/apg/patterns/alert/

Reply:
Yes, referenced this, could you please review the section and add thoughts

Reply:
Yes the a11y seems to be fine. 💯